clusterloader2/testing/load/modules/scheduler-throughput.yaml

```diff
@@ -0,0 +1,123 @@
+{{$namespaces := .namespaces}}
+{{$schedulerThroughputNamespaces := .schedulerThroughputNamespaces}}
+{{$schedulerThroughputPodsPerDeployment := .schedulerThroughputPodsPerDeployment}}
+
+{{$ENABLE_HUGE_SERVICES := DefaultParam .CL2_ENABLE_HUGE_SERVICES false}}
+{{$EXIT_AFTER_EXEC := DefaultParam .CL2_EXIT_AFTER_EXEC false}}
+{{$IS_SMALL_CLUSTER := lt .Nodes 100}}
```

Comment:
I would prefer to pass this as parameter so we don't have duplicated logic in two places that specifies what small cluster is.

Reply:
Not necessary anymore